pkg/catalog/catalog.go

```diff
@@ -259,7 +257,7 @@ func CreateContent(ctx context.Context, secretAuth []byte, bundleImage string) (
 
 // generateFBCContent creates a File-Based Catalog using the bundle image and context
 func generateFBCContent(ctx context.Context, f *fbcutil.FBCContext, bundleImage string) (string, error) {
-	log.Infof("Creating a File-Based Catalog of the bundle %q", bundleImage)
+	logrus.Infof("Creating a File-Based Catalog of the bundle %q", bundleImage)
```

Comment:
Why bother with `logrus` at all here?  We're using `klog` everywhere else in this same file